coreneuron/CMakeLists.txt

```diff
@@ -144,7 +198,11 @@ macro(mod2c_from_file name modlist searchpath)
             else()
                 list(APPEND added_mod_files "${modpath_}")
                 list(APPEND mod_filenames "${filename}")
-                mod2c_target(${name} "${modpath_}")
+                if(${ENABLE_ISPC_TARGET})
+                    mod2c_target(KEY ${name} MODFILE "${modpath_}" TARGET ispc)
```

Comment:
The pattern below is seen at least 3 times:

```
if(${ENABLE_ISPC_TARGET})
    mod2c_target(KEY ${name} MODFILE "${GAPMOD}" TARGET ispc)
else()
    mod2c_target(KEY ${name} MODFILE "${GAPMOD}" TARGET serial)
endif()
```

maybe you can 
```
if(${ENABLE_ISPC_TARGET})
    set(TARGET iscpc)
else()
    set(TARGET serial)
endif()
```

And then only do: `mod2c_target(KEY ${name} MODFILE "${GAPMOD}" TARGET ${TARGET})`